Library/SharedFunctions.swift

```diff
@@ -216,3 +216,27 @@ internal func classNameWithoutModule(_ class: AnyClass) -> String {
     .dropFirst()
     .joined(separator: ".")
 }
+
+typealias FormattedPledgeParameters = (pledgeTotal: String, rewardId: String?, locationId: String?)
```

Comment:
When I read `FormattedPledgeParameters` I think about this going through a formatter, I'm wondering if `SanitizedPledgeParameters` or `SanitizedPledgeParams` describes what we're doing more accurately?